### PR TITLE
drivers: dai: ssp: fix mdivr register for ACE platform

### PR DESCRIPTION
**Pull request by abonislawski**
Set correct offsets for CAVS&ACE

### CODE REVIEW

File: drivers/dai/intel/ssp/ssp.h [hunk 1]
```diff
@@ -223,14 +223,19 @@
 #define SHIM_CLKCTL_I2SFDCGB(x)		BIT(20 + x)
 #define SHIM_CLKCTL_I2SEFDCGB(x)	BIT(18 + x)
 
+#if CONFIG_INTEL_ADSP_CAVS
```
Comment (stephanosio) [inline drivers/dai/intel/ssp/ssp.h:226]:
```suggestion
#ifdef CONFIG_INTEL_ADSP_CAVS
```

**Review by stephanosio — DISMISSED** [review]

**Review by juimonen — COMMENTED** [review]
So I had some differences for CAVS15 and 25 so if you want to separate ACE changes, then preferably:
#ifdef CONFIG_SOC_SERIES_INTEL_ACE

otherwise fine with me.

**Comment by abonislawski** [comment]
Updated with ifdef and CONFIG_SOC_SERIES_INTEL_ACE

**Review by juimonen — APPROVED** [review]
lgtm

**Review by marcinszkudlinski — APPROVED** [review]

**Pull request merged**